Providers/Modules/WLI/plugins/common/in_wlm_discovery.rb

```diff
@@ -115,6 +119,25 @@ def get_last_discovery_time()
       end # begin
     end # method get_last_discovery_time
 
+    def get_vm_metadata()
+      begin
+        url_metadata="http://169.254.169.254/metadata/instance?api-version=#{@metadata_api_version}"
```

Comment:
>169.254.169.254 [](start = 29, length = 15)

why are you using a hardcoded ip address here?

Reply:
Thanks, Sounds good!

---
In reply to: [163019847](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/353#discussion_r163019847) [](ancestors = 163019847)